docs/pages/eas-update/publish.mdx

```diff
@@ -39,14 +39,14 @@ The above command runs `npx expo export` behind the scenes to generate a **dist*
   Icon={LayersTwo02Icon}
 />
 
-## Test a published update
+## Testing a published update
 
-After the update is uploaded to EAS, you can use one of the following methods to test the update:
+After the update is uploaded to EAS Update, you can use one or more of the following methods to test the update:
```

Comment:
```suggestion
After the update is uploaded to EAS Update, you can use one of the following methods to test the update:
```

At a single given time, one user can only use one method so I think the later of the this sentence is okay.